.gitignore

```diff
@@ -1,7 +1,8 @@
-/nbproject/
-/target/
-/Playlists/
-/test/
-*.json
-*.txt
-nb*.xml
\ No newline at end of file
+/nbproject/
+/target/
+/Playlists/
+/test/
+*.json
+*.txt
+nb*.xml
+.idea
```

Comment:
It'd be ideal to not change the .gitignore, as it sorta falls out of scope for this PR.